packages/ng-table-builder/src/table-builder.component.ts

```diff
@@ -307,6 +308,15 @@ export class TableBuilderComponent<T>
         }
     }
 
+    public updateColumnsSchema(patch: PlainObjectOf<Partial<ColumnsSchema>>): void {
+        this.templateParser.updateColumnsSchema(patch);
+        this.ngZone.runOutsideAngular((): void => {
+            window.requestAnimationFrame((): void => {
```

Comment:
can we use `changeSchema` without requestAnimationFrame?

Reply:
yas, i'll do that